src/cdi_analytic/Analytic_MultigroupOpacity.cc

```diff
@@ -29,26 +29,26 @@ namespace rtt_cdi_analytic {
  * number of Analytic_Opacity_Model objects given in the models argument must
  * be equal to the number of groups.
  *
- * \param[in] groups vector containing the group boundaries in keV from lowest 
+ * \param[in] groups vector containing the group boundaries in keV from lowest
  *               to highest
  * \param[in] reaction_in rtt_cdi::Reaction type (enumeration)
  * \param[in] model_in CDI type
  */
-Analytic_MultigroupOpacity::Analytic_MultigroupOpacity(
-    const sf_double &groups, rtt_cdi::Reaction reaction_in,
-    rtt_cdi::Model model_in)
-    : group_boundaries(groups), reaction(reaction_in), model(model_in) {
+Analytic_MultigroupOpacity::Analytic_MultigroupOpacity(sf_double groups,
+                                                       rtt_cdi::Reaction reaction_in,
+                                                       rtt_cdi::Model model_in)
+    : group_boundaries(std::move(groups)), reaction(reaction_in), model(model_in) {
```

Comment:
another move question

Reply:
I don't like that `std::vector<double>` is the somewhat cryptic "sf_double", but I'll live